src/components/DesktopPatientTable/index.js

```diff
@@ -105,17 +105,24 @@ const DesktopPatientTable = (props) => {
                     <Status selectedCases={patient.status} />
                     <Link
                       className="card-name patient-link--small min-width-20 mr-2"
-                      to={routes.createEncounter.path.replace(
+                      to={routes.editPatient.path.replace(
                         ':patientId',
                         patient.patientId,
                       )}>
                       {patient.fullName}
                     </Link>
                   </>
                 ) : (
-                  <p className="card-name patient-link--small min-width-20 mr-2 mb-0">
-                    {patient.fullName}
-                  </p>
+                  <Link
```

Comment:
please move Link from the condition: leave Status in it only, place Link below after condition is closed